src/Sulu/Component/Webspace/Analyzer/Attributes/WebsiteRequestProcessor.php

```diff
@@ -70,6 +86,11 @@ function (PortalInformation $a, PortalInformation $b) {
 
         /** @var PortalInformation $portalInformation */
         $portalInformation = reset($portalInformations);
+        foreach ($portalInformations as $information) {
```

Comment:
i have played around with the priorities. i think it should work now only with the usort.